schematypes.go

```diff
@@ -1488,101 +1509,196 @@ type DisassembledInstruction struct {
-func (m *PauseResponse) GetSeq() int                   { return m.Seq }
-func (m *StackTraceRequest) GetSeq() int               { return m.Seq }
-func (m *StackTraceResponse) GetSeq() int              { return m.Seq }
-func (m *ScopesRequest) GetSeq() int                   { return m.Seq }
-func (m *ScopesResponse) GetSeq() int                  { return m.Seq }
-func (m *VariablesRequest) GetSeq() int                { return m.Seq }
-func (m *VariablesResponse) GetSeq() int               { return m.Seq }
-func (m *SetVariableRequest) GetSeq() int              { return m.Seq }
-func (m *SetVariableResponse) GetSeq() int             { return m.Seq }
-func (m *SourceRequest) GetSeq() int                   { return m.Seq }
-func (m *SourceResponse) GetSeq() int                  { return m.Seq }
-func (m *ThreadsRequest) GetSeq() int                  { return m.Seq }
-func (m *ThreadsResponse) GetSeq() int                 { return m.Seq }
-func (m *TerminateThreadsRequest) GetSeq() int         { return m.Seq }
-func (m *TerminateThreadsResponse) GetSeq() int        { return m.Seq }
-func (m *ModulesRequest) GetSeq() int                  { return m.Seq }
-func (m *ModulesResponse) GetSeq() int                 { return m.Seq }
-func (m *LoadedSourcesRequest) GetSeq() int            { return m.Seq }
-func (m *LoadedSourcesResponse) GetSeq() int           { return m.Seq }
-func (m *EvaluateRequest) GetSeq() int                 { return m.Seq }
-func (m *EvaluateResponse) GetSeq() int                { return m.Seq }
-func (m *SetExpressionRequest) GetSeq() int            { return m.Seq }
-func (m *SetExpressionResponse) GetSeq() int           { return m.Seq }
-func (m *StepInTargetsRequest) GetSeq() int            { return m.Seq }
-func (m *StepInTargetsResponse) GetSeq() int           { return m.Seq }
-func (m *GotoTargetsRequest) GetSeq() int              { return m.Seq }
-func (m *GotoTargetsResponse) GetSeq() int             { return m.Seq }
-func (m *CompletionsRequest) GetSeq() int              { return m.Seq }
-func (m *CompletionsResponse) GetSeq() int             { return m.Seq }
-func (m *ExceptionInfoRequest) GetSeq() int            { return m.Seq }
-func (m *ExceptionInfoResponse) GetSeq() int           { return m.Seq }
-func (m *ReadMemoryRequest) GetSeq() int               { return m.Seq }
-func (m *ReadMemoryResponse) GetSeq() int              { return m.Seq }
-func (m *DisassembleRequest) GetSeq() int              { return m.Seq }
-func (m *DisassembleResponse) GetSeq() int             { return m.Seq }
+func (m *Request) GetSeq() int                                    { return m.Seq }
+func (m *Event) GetSeq() int                                      { return m.Seq }
+func (m *Response) GetSeq() int                                   { return m.Seq }
+func (m *ErrorResponse) GetSeq() int                              { return m.Seq }
+func (r *ErrorResponse) GetResponse() *Response                   { return &r.Response }
```

Comment:
This section is kind of a mess now. Might be a good time to follow the example of some of the standard libraries (e.g. https://golang.org/src/io/io.go#L439) and specify the methods after the receiver types.